pkg/flink/handler.go

```diff
@@ -92,11 +93,12 @@ func NewFlinkTaskContext(ctx context.Context, taskCtx FlinkTaskExecContext) (*Fl
 	}
 
 	return &FlinkTaskContext{
-		ClusterName: cn,
-		Namespace:   taskMetadata.GetNamespace(),
-		Annotations: GetDefaultAnnotations(taskMetadata),
-		Labels:      GetDefaultLabels(taskMetadata),
-		Job:         job,
+		ClusterName:          cn,
+		Namespace:            taskMetadata.GetNamespace(),
+		Annotations:          GetDefaultAnnotations(taskMetadata),
+		Labels:               GetDefaultLabels(taskMetadata),
+		EnvironmentVariables: GetDefaultEnvVars(taskMetadata),
```

Comment:
```suggestion
		EnvironmentVariables: GetEnvironmentVariables(taskMetadata),
```
Sorry if my comment was confusing we can leave like you add before 😄

Reply:
Oh yeah... I misunderstood it.. sorry! 🙈